systems/notification/mailer/pkg/db/mailer_repo.go

```diff
@@ -0,0 +1,24 @@
+package db
+
+import (
+	"github.com/ukama/ukama/systems/common/sql"
+)
+
+type MaillingRepo interface {
+	SendEmail(mail *Mailing) error
+}
+
+type maillingRepo struct {
+	Db sql.Db
+}
+
+func NewMaillingRepo(db sql.Db) MaillingRepo {
+	return &maillingRepo{
+		Db: db,
+	}
+}
+
+func (s *maillingRepo) SendEmail(mail *Mailing) error {
```

Comment:
Should we have way to pull communication based on certain criteria like email or userid ? It's totally fine to skip the implemenation now and have it in future. Just need to make sure we have that in issue list.

Reply:
Make sense .I will add a method to pull email by ID.